apps/mobile/src/components/SetContactsScreen/components/ContactListSelect/index.tsx

```diff
@@ -44,6 +45,7 @@ function ContactsListSelect({
   useEffect(() => {
     return () => {
       resolveAllContactsAsSeen()
+      void areContactsPermissionsGranted()
```

Comment:
Nothing done with the result. Should the screen react to contacts not being grated here?